cmd/cluster-agent/app/app.go

```diff
@@ -194,6 +198,27 @@ func start(cmd *cobra.Command, args []string) error {
 		log.Debugf("Health check listening on port %d", healthPort)
 	}
 
+	// Attempt to create and start the core remote config service, creating the local client only if that succeeds
+	var configService *remoteconfig.Service
+	var rcClient *remote.Client
+	if config.Datadog.GetBool("remote_configuration.enabled") {
+		configService, err = remoteconfig.NewService()
+		if err == nil {
+			if err := configService.Start(context.Background()); err == nil {
```

Comment:
```suggestion
			if err := configService.Start(mainCtx); err == nil {
```
I assume `configService.Start` handles a graceful shutdown since it accepts a context?